core/src/test/scala/org/broadinstitute/dsde/rawls/snapshot/SnapshotServiceSpec.scala

```diff
@@ -152,102 +150,7 @@ class SnapshotServiceSpec extends AnyWordSpecLike with Matchers with MockitoSuga
-
-      val mockWorkspaceManagerDAO = mock[WorkspaceManagerDAO](RETURNS_SMART_NULLS)
-
-      when(mockWorkspaceManagerDAO.createDataRepoSnapshotReference(any[UUID], any[UUID], any[DataReferenceName], any[Option[DataReferenceDescriptionField]], any[String], any[CloningInstructionsEnum], any[OAuth2BearerToken]))
-        .thenReturn(new DataRepoSnapshotResource().metadata(new ResourceMetadata().resourceId(UUID.randomUUID()).workspaceId(UUID.randomUUID()).name("foo").description("").cloningInstructions(CloningInstructionsEnum.NOTHING)).attributes(new DataRepoSnapshotAttributes()))
-
-      when(mockWorkspaceManagerDAO.createBigQueryDatasetReference(any[UUID], any[ReferenceResourceCommonFields], any[GcpBigQueryDatasetAttributes], any[OAuth2BearerToken])).thenThrow(new RuntimeException)
-
-      val workspace = minimalTestData.workspace
-      val deltaLayer = new DeltaLayer(mockBigQueryServiceFactory, new MockDeltaLayerWriter, mockSamDAO, fakeRawlsClientEmail, fakeDeltaLayerStreamerEmail)
-
-      val snapshotService = SnapshotService.constructor(
-        slickDataSource,
-        mockSamDAO,
-        mockWorkspaceManagerDAO,
-        deltaLayer,
-        "fake-terra-data-repo-dev",
-        fakeRawlsClientEmail,
-        fakeDeltaLayerStreamerEmail
-      )(userInfo)
-
-      val createException = intercept[RawlsExceptionWithErrorReport] {
-        Await.result(snapshotService.createSnapshot(workspace.toWorkspaceName, NamedDataRepoSnapshot(DataReferenceName("foo"), Option(DataReferenceDescriptionField("foo")), UUID.randomUUID())), Duration.Inf)
-      }
-      createException.errorReport.statusCode shouldBe Some(StatusCodes.InternalServerError)
-      assert(createException.errorReport.message.contains("Unable to create snapshot reference in workspace"))
-
-      verify(mockSamDAO, times(1)).listPoliciesForResource(any[SamResourceTypeName], any[String], any[UserInfo])
-      verify(mockBigQueryServiceFactory, times(2)).getServiceForProject(workspace.googleProject)
-      verify(mockWorkspaceManagerDAO, times(1)).createDataRepoSnapshotReference(any[UUID], any[UUID], any[DataReferenceName], any[Option[DataReferenceDescriptionField]], any[String], any[CloningInstructionsEnum], any[OAuth2BearerToken])
-      verify(mockWorkspaceManagerDAO, times(1)).createBigQueryDatasetReference(any[UUID], any[ReferenceResourceCommonFields], any[GcpBigQueryDatasetAttributes], any[OAuth2BearerToken])
-
-      // see SnapshotService line 65, "fire and forget these undos, we've made our best effort to fix things at this point"
-      // because the call to workspaceManagerDAO.deleteDataRepoSnapshotReference is fire-and-forget, we can't count on it happening
-      // by this point of this test. We need an eventually here:
-      eventually {
-        verify(mockWorkspaceManagerDAO, times(1)).deleteDataRepoSnapshotReference(any[UUID], any[UUID], any[OAuth2BearerToken])
-      }
-
-    }
```

Comment:
this test is no longer needed - since we don't create a WSM dataset reference for the companion, there's no way for the WSM dataset reference creation to fail, making this test invalid.